### PR TITLE
Changed SimplePriceOracle so ETH price can be set on testnets

### PR DESCRIPTION
**Pull request by ajb413**
Changes for the Ropsten testnet price feed. ETH price can now be set.

```
saddle deploy -n ropsten SimplePriceOracle
saddle verify -n ropsten $ETHERSCAN_API_KEY 0x7BBF806F69ea21Ea9B8Af061AD0C1c41913063A1 SimplePriceOracle
```

### CODE REVIEW

**Review by coburncoburn — APPROVED** [review]

File: tests/Lens/CompoundLensTest.js [hunk 1]
```diff
@@ -204,7 +204,7 @@ describe('CompoundLens', () => {
       ).toEqual(
         {
           cToken: cEth._address,
-          underlyingPrice: "1000000000000000000",
+          underlyingPrice: "0",
```
Comment (torreyatcitty) [inline tests/Lens/CompoundLensTest.js:207]: Can we add a test that sets the underlying price of Eth so we can verify it goes non-zero after that happens?

**Review by alexanderramos89 — APPROVED** [review]

**Review by jflatow — APPROVED** [review]
